docs/release/trg-0/trg-7-1.md

```diff
@@ -9,12 +9,103 @@ Proposed release date: "mandatory after": 19th of May 2023
 
 | Status | Created     | Post-History  |
 |--------|-------------|---------------|
-| Draft  | 24-Feb-2022 | n/a           |
+| Draft  | 13-Apr-2023 | Moved from OSS Development           |
 
 ## Why
 
-Describe in this section why the .
+Eclipse Tractus-X is an open source project hosted by the Eclipse Foundation licensed under the [Apache License 2.0](https://spdx.org/licenses/Apache-2.0). The legal obligations of the content must be observed in all forms of which the content is available.
+
+This page contains information about legal documentation requirements in your repositories. The reference is always the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#legaldoc).
+
+:::info
+
+The requirements described here must be met for each contribution.
+
+:::
 
 ## Description
 
-Describe the TRG on how.
+The following files have to be added at root level of your Eclipse Tractus-X repository:
```

Comment:
perhaps have to also bold or must: "must be part of your repository root folder"